### DISCUSSION TITLE
Double input tidak tampil di menu 2

### DISCUSSION THREAD

**Discussion by reyhanpurnawan** [teamalgoritma/community#428] [Q&A] [answered]
[Shiny CP.zip](https://github.com/teamalgoritma/community/files/9897888/Shiny.CP.zip)

**Comment by kinantytasyao** [comment, marked as answer]
Dear @reyhanpurnawan ,

Double input sebenarnya muncul, namun urutan munculnya adalah sebagai berikut: selectInput - plot -  checkboxGroupInput. Hal ini karena kode Bapak di ui,R urutannya juga seperti itu:

`
tabItem(
          tabName = "menu_2",
          fluidRow(box(width = 12,
                       selectInput(inputId = "select_date",
                                   label = "Choose a date:",
                                   choices = unique(df_clean$date)))),
`
`
fluidRow(
            box(width = 12,
                plotlyOutput(outputId = "plot_1"))),
`
`
fluidRow(box(width = 12,
                       checkboxGroupInput(inputId = "select_country",
                                          label = "Choose a country",
                                          choices = unique(df_clean$name),
                                          selected = unique(df_clean$name))))
    )
`

Sehingga Bapak mungkin bisa mengubah urutan kode di ui.R jika ingin double input tidak terpisah plot.
Demikian, semoga dapat membantu:)

**Reply by reyhanpurnawan** [reply]
<img width="1440" alt="Screen Shot 2022-10-31 at 15 40 55" src="https://user-images.githubusercontent.com/88662517/198966883-0abc006c-9611-4e25-9dbf-40671d34eb4c.png">
Terima kasih mba responnya, maksud saya error seperti ini, tidak ada output

**Reply by kinantytasyao** [reply]
Pada file Server.R di line 11 kode Bapak adalah:
`
filter(name == input$select_country)
`

Karena menggunakan checkboxGroupInput kita ingin membandingkan lebih dari 1 kategori untuk suatu plot dan seharusnya menggunakan tanda `%ini%` bukan `==`, sehingga kodenya menjadi:

`
filter(name %in% input$select_country)
`

Demikian yang dapat kami sampaikan. Semoga membantu.

**Reply by reyhanpurnawan** [reply]
Baik mba mantap. Terima kasih banyak
